src/SFML/Graphics/Font.cpp

```diff
@@ -143,6 +144,15 @@ bool Font::loadFromFile(const std::string& filename)
         return false;
     }
 
+    // Load the stroker that will be used to outline the font
+    FT_Stroker stroker;
+    if (FT_Stroker_New(static_cast<FT_Library>(m_library), &stroker) != 0)
```

Comment:
```cleanup``` is always called in the destructor and ```loadFromFile(...)```. I didn't change any of the program flow regarding the cleanup function so if there is a problem with it then it has been in the code base for quite some time. ;)